proxy/proxy.go

```diff
@@ -1009,3 +1011,12 @@ func (p *Proxy) Close() error {
 	close(p.quit)
 	return nil
 }
+
+func (p *Proxy) setCommonSpanInfo(u *url.URL, r *http.Request, s ot.Span) {
+	ext.Component.Set(s, "skipper")
```

Comment:
This seems unnecessary/redundant with the service name or component name provided initially to the Tracers. I'd use it if we intend to have more fine-grained components like `skipper:proxy`, `skipper:dataclient` or similar.

Reply:
I think that `component` tag doesn't specify what is the nature of the application but the technology itself, your tracer could be named `skipper`, `skipper-proxy`, `skipper-myapp`... but the app is a `skipper`.

From the [conventions](https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table)
> component (string): The software package, framework, library, or module that generated the associated Span. E.g., "grpc", "django", "JDBI".